.pylintrc

```diff
@@ -290,7 +290,7 @@ spelling-store-unknown-words=no
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=120
```

Comment:
I've been thinking that maybe 120 is too much. In cases where you want to have two files side by side, even in PyCharm at slightly enlarged font (like mine), I can't comfortably fit two 120 lines side by side. 100 fits though... what do you think? Also, Python PEP8 suggests 100 as the max: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
I like 120 but don't particularly prefer it to 100.

Reply:
I like 120 in general, but on the rare occasion that you want to have two files open at the same time, it helps a lot to have slightly more reasonable lines.

Actually, since you use the PyCharm git tools, I'd think you would prefer the shorter line length too since PyCharm shows diffs side by side.